src/map/battle.cpp

```diff
@@ -6978,7 +6978,7 @@ struct Damage battle_calc_magic_attack(struct block_list *src,struct block_list
 				}
 				break;
 			case NPC_DARKBREATH:
-				ad.damage = tstatus->hp * (skill_lv <= 5 ? 100 / (2 * 6 - skill_lv) : 50) / 100;
+			        ad.damage = tstatus->hp * (skill_lv <= 5 ? 100 / (2 * (6 - skill_lv)) : 50) / 100;
```

Comment:
```suggestion
				ad.damage = tstatus->hp * (skill_lv <= 5 ? 100 / (2 * (6 - skill_lv)) : 50) / 100;
```
Remove <SPACE> and add <TAB>